File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/VehicleResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class VehicleResponse 
{
    public string Id { get; set; }
    public string Name { get; set; }
    public long Capacity { get; set; }
    public GeoResponse Depot { get; set; }
    public long StartWindow { get; set; }
    public long EndWindow { get; set; }
    public GeoResponse Current { get; set; }
    public StopResponse DepotAsStop { get; set; }
}

File: server-scala-play-server/app/model/VehicleResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for VehicleResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-11T19:31:43.310890579Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class VehicleResponse(
  id: Option[String],
  name: Option[String],
  capacity: Option[Long],
  depot: Option[GeoResponse],
  startWindow: Option[Long],
  endWindow: Option[Long],
  current: Option[GeoResponse],
  depotAsStop: Option[StopResponse]
)

object VehicleResponse {
  implicit lazy val vehicleResponseJsonFormat: Format[VehicleResponse] = Json.format[VehicleResponse]
}

